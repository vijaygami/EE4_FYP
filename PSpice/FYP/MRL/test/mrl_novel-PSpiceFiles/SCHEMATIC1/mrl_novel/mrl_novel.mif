 lib="C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\mrl_novel-pspicefiles\mrl_novel.lib", offset=97, size=55
 lib="C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\mrl_novel-pspicefiles\mrl_novel.lib", offset=42, size=54
