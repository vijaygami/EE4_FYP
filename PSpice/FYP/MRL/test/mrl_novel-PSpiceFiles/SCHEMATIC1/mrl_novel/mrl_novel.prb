[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL NEVER
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORTICKS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE V(T4:d)
MARKERID 197
TRACEADDEXT
END TRACE V(T4:d)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
