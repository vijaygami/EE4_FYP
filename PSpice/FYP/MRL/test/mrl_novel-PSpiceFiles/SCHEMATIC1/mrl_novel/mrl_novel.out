
**** 06/18/16 20:40:43 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-mrl_novel"  [ C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\mrl_novel-pspicefiles\schematic1\mrl_novel.sim


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "mrl_novel.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../mrl_novel-pspicefiles/mrl_novel.lib" 
.LIB "../../../mrl_novel-pspicefiles/vteam.lib" 
* From [PSPICE NETLIST] section of C:\Users\Vijay\Documents\PSpice\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 0.06m 0 1n 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source MRL_NOVEL
V_Vdd         VDD 0 5Vdc
V_Vin1         VIN1 0   PWL   
+ FILE "C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\v1.txt" 
M_T4         N01597 VIN1 VDD VDD Mbreakp           
M_M3         N01597 VIN1 VIN1 0 Mbreakn           

**** RESUMING mrl_novel.cir ****
.END

**** 06/18/16 20:40:43 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-mrl_novel"  [ C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\mrl_novel-pspicefiles\schematic1\mrl_novel.sim


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               Mbreakp         Mbreakn         
               PMOS            NMOS            
       LEVEL    1               1            
           L  250.000000E-09  250.000000E-09 
           W    6.300000E-06    2.500000E-06 
         VTO   -1.5             1            
          KP   20.000000E-06   50.000000E-06 
       GAMMA    0               0            
         PHI     .6              .6          
      LAMBDA    0               0            
          IS   10.000000E-15   10.000000E-15 
          JS    0               0            
          PB     .8              .8          
        PBSW     .8              .8          
          CJ    0               0            
        CJSW    0               0            
        CGSO    0               0            
        CGDO    0               0            
        CGBO    0               0            
         TOX    0               0            
          XJ    0               0            
       UCRIT   10.000000E+03   10.000000E+03 
      DIOMOD    1               1            
         VFB    0               0            
        LETA    0               0            
        WETA    0               0            
          U0    0               0            
        TEMP    0               0            
         VDD    5               5            
       XPART    0               0            


**** 06/18/16 20:40:43 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-mrl_novel"  [ C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\mrl_novel-pspicefiles\schematic1\mrl_novel.sim


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  VDD)    5.0000  ( VIN1)    0.0000  (N01597)    5.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_Vdd       -5.010E-12
    V_Vin1       0.000E+00

    TOTAL POWER DISSIPATION   2.51E-11  WATTS

Reducing minimum delta to make the circuit converge.
Reducing minimum delta to make the circuit converge.


          JOB CONCLUDED

**** 06/18/16 20:40:43 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-mrl_novel"  [ C:\Users\Vijay\Google Drive\PSpice\FYP\MRL\test\mrl_novel-pspicefiles\schematic1\mrl_novel.sim


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  License check-out time            =       18.24
  Total job time (using Solver 1)   =         .39
